app/models/solidus_paypal_commerce_platform/payment_source.rb

```diff
@@ -3,8 +3,19 @@
 module SolidusPaypalCommercePlatform
   class PaymentSource < SolidusSupport.payment_source_parent_class
     self.table_name = "paypal_commerce_platform_sources"
+    enum paypal_funding_source: {
```

Comment:
Do we need a presence validator for it?

Reply:
Seeing as it's not a vital attribute, I think perhaps we can skip it? It then means the old records wont become invalid as well.

Reply:
👍